addon/styles/chart.less

```diff
@@ -7,196 +7,40 @@
     height: 100%;
     width: 100%;
 
-    .chart-header,
-    .axis-title,
-    .x.axis {
-        flex-grow: 0;
-        flex-shrink: 0;
-    }
-
-    .chart-canvas {
-        flex-grow: 1;
-        flex-shrink: 1;
-
-        overflow-x: hidden;
-        overflow-y: auto;
-        width: 100%;
-    }
-
-    .chart-canvas svg {
-        display: block;
-    }
-
-    .axis-title {
-        font-weight: bold;
+    .axis line,
+    .axis path {
+        fill: none;
+        stroke: @alto;
+        stroke-width: 1px;
+        shape-rendering: crispEdges;
     }
 
-    .axis {
+    .axis,
+    .tick {
+        overflow: visible;
         text {
             fill: @silver-chalice;
         }
     }
 
     .x.axis {
+        flex-grow: 0;
+        flex-shrink: 0;
         margin-bottom: 5px;
-        overflow: visible;
         width: 100%;
     }
 
-    .x.axis line,
-    .x.axis path {
-        fill: none;
-        stroke: @alto;
-        stroke-width: 1px;
-        shape-rendering: crispEdges;
-    }
-
-    .axes {
-        overflow: visible;
-        opacity: 1;
-    }
-
-    .y.axis,
-    .yr.axis {
-        overflow: visible;
-        opacity: 1;
-    }
-
-    .y.axis line,
-    .y.axis path,
-    .yr.axis line,
-    .yr.axis path {
-        fill: none;
-        stroke: @alto;
-        stroke-width: 1px;
-        shape-rendering: crispEdges;
-    }
-
-    .grid-line {
-        fill: none;
-        stroke: @silver;
-        opacity: .5;
-        shape-rendering: crispEdges;
-    }
-
-    .grid-line line {
-        fill: none;
-        stroke: @silver;
-        opacity: .5;
-        shape-rendering: crispEdges;
-    }
-
-    .brush rect.background {
-        z-index: -999;
-    }
-
-    .brush rect.extent {
-        fill: steelblue;
-        fill-opacity: .125;
-    }
-
-    .brush .resize path {
-        fill: @gallery;
-        stroke: @dove-gray;
+    .chart-not-available {
+        text-shadow:
+            -1px -1px 0 #fff,  
```

Comment:
Super minor but these rows seem a little, spacing-wise. I think this one has 12 spaces and the 3 below have 13.